### PR TITLE
Browserify

### PR DESCRIPTION
**Pull request by switters**
Maybe related to #8

Needs browserify fields in package.json so unused files/dependencies aren't being bundled in.

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -5,11 +5,13 @@
 
 var isBrowser = ('undefined' !== typeof window)
 
+// server specific
 if (!isBrowser) {
   var axon = require('axon')
   var os = require('os')
   var host = os.hostname()
 }
+// broswer specific
```
Comment (switters) [inline lib/index.js]: Define: broswer - a browser that often says 'dude'.

**Comment by switters** [comment]
As axon and os were the packages not necessary in the browser, I flagged them in the package.json as false. This fix saved us more than two thirds of our compiled loc (~5000 lines removed).

**Comment by switters** [comment]
@lvivier - review?

File: lib/index.js [hunk 2]
```diff
@@ -32,9 +34,10 @@ if(!isBrowser) {
 
 
 /**
- * Logger builder
+ * Build Logger
+ *
  * @param {String} ctx
- * @param {Array} [addrs]
+ * @api public
```
Comment (lvivier) [inline lib/index.js]: any specific reason to add all these lines? jsdoc considers any function not marked as private to be implicitly public fyi

**Comment by lvivier** [comment]
:+1:

File: lib/index.js (hunk 2 above)
Comment (switters) [inline lib/index.js]: hm, then no

**Pull request merged**